src/formatDate.js

```diff
@@ -8,7 +8,49 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  // '2020-02-18', ['YYYY', 'MM', 'DD', '-'], ['DD', 'MM', 'YY', '.']
+
+  const separatorFrom = fromFormat[fromFormat.length - 1]; // '-',
+  const separatorToFormat = toFormat[toFormat.length - 1]; // '.'
+
+  const formatedYear = [];
+
+  const newDate = date.split(separatorFrom); // ['2020', '02', '18'],
```

Comment:
The variable `newDate` may be better named as `dateParts` to clearly indicate that it contains the parts of the date after splitting.